### ISSUE TITLE
vichan api for lapis-chan

### ISSUE THREAD

**Issue by ghost** [karai17/lapis-chan#41]
If I see that correctly, lapis-chan doesn't provide an API at this stage. Can we have a vichan-like API please?

ref: https://github.com/vichan-devel/vichan-API

**Comment by shakesoda** [comment]
I've mentioned needing an API a bunch of times to @karai17 already, it'd be useful for issues like #15 as well (if not a prerequisite).

:+1:

**Comment by josefnpat** [comment]
:+1:

**Comment by karai17** [comment]
Yeah, an API is definitely something I plan to work on at some point. Forking the vichan API (which was a fork from 4chan, interestingly enough) is probably something worth looking into.

**Comment by karai17** [comment]
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api

Some useful information about designing an API here.
